main.cpp

```diff
@@ -4630,6 +4658,27 @@ vector<uint8_t> sign_guts_bin(iostream_memory_access in, private_t private_key,
         new_block.items.push_back(version);
     }
 
+    // Add entry point when signing Arm images
```

Comment:
Hmm; i hadn't really considered the BIN case where we don't know where the user is planning to load it. Perhaps we don't do all this and just give a warning instead? (or allow a "target address" option) - i'd be happy with the former for now, we can always add the new option later if it is useful

Reply:
I've changed it to use `bin_start`, which already comes from the `--offset` option that you can specify on the command line